x/leverage/keeper/keeper.go

```diff
@@ -65,11 +68,6 @@ func (k Keeper) LendAsset(ctx sdk.Context, lenderAddr sdk.AccAddress, loan sdk.C
 		return sdkerrors.Wrap(types.ErrInvalidAsset, loan.String())
 	}
 
-	if !k.bankKeeper.HasBalance(ctx, lenderAddr, loan) {
```

Comment:
just a note @toteki, there isn't a need to check this as `SendCoinsFromAccountToModule` will error if there is an insufficient balance.